words/words.md

```diff
@@ -28,15 +28,15 @@ Sicher hortest Du kein Toilettenpapier - aber wenn Politiker die Angst selbst f
 
 Wir (Marcel, Epidemiologe + Nicky, Kunst / Code) sind ehrlich gesagt auch besorgt. Und wir wetten, dass Du es auch bist! Deshalb haben wir unsere Angst dazu genutzt, um diese **interaktive Simulationen** zu erstellen. Unser Ziel ist, dass Du nicht in Panik verharrst, sondern versuchst zu verstehen und zu lernen:
 
-* **Über die letzten paar Monate** (Epidemiologie 101, SEIR-Modell, R & R0)
-* **Über die nächsten paar Monate** (Sperren, Kontaktverfolgung, Masken)
+* **Über die letzten paar Monate** (1x1 der Epidemiologie 101, SEIR-Modell, R & R0)
+* **Über die nächsten paar Monate** (Lockdowns, Contact Tracing, Masken)
 * **Über die nächsten paar Jahre** (Verlust der Immunität? Kein Impfstoff?)
 
-Diese interaktive Simulation (veröffentlicht am 1. Mai 2020. Siehe Fußnote!→[^timestamp]) soll Dir zugleich Hoffnung geben und Angst machen. Denn um COVID-19 auf eine Weise zu schlagen, die auch **unsere geistige Gesundheit und unsere finanzielle Situation schützt**, brauchen wir zum einen Optimismus, um Pläne zu machen, zum anderen Pessimismus, um Backup-Pläne zu erstellen. Wie Gladys Bronwyn Stern einmal sagte: *"Der Optimist erfindet das Flugzeug und der Pessimist den Fallschirm."*
+Diese interaktive Simulation (veröffentlicht am 1. Mai 2020. Siehe Fußnote!→[^timestamp]) soll Dir zugleich Hoffnung geben und Angst machen. Denn um COVID-19 auf eine Weise zu schlagen, die auch **unsere geistige Gesundheit und unsere finanzielle Lage schützt**, brauchen wir zum einen Optimismus, um einen Plan zu machen, zum anderen Pessimismus, um einen Plan B zu entwickeln. Wie Gladys Bronwyn Stern einmal sagte: *"Der Optimist erfindet das Flugzeug und der Pessimist den Fallschirm."*
```

Comment:
Im Original ist das Wortpaar "plans" und "backup plans". Würde versuchen die parallele Struktur in der Übersetzung stilistisch zu erhalten. "Plan B" ist sicherlich gängiger als "Backup-Plan", hat aber leider keinen Plural und der Plural ist hier inhaltlich wichtig. Es geht nicht nur darum nur einen Plan und einen Plan B zu machen.

Vorschlag: Würde bei "Pläne" und "Backup-Pläne" bleiben.

Reply:
Verstehe den Punkt, aber geht die Wichtigkeit des Plurals nicht auch aus dem ganzen Projekt an sich hervor? Ich denke in dem Fall eher vom Lesefluss her: Lassen vertraute Formulierungen wie "Plan" und "Plan B" deutschsprachige Lesende eher am Ball bleiben als weniger vertraute wie "Pläne" und "Backup-Pläne"?

Reply:
Hier würde ich mich @TQueV anschließen. Man merkt sonst sehr schnell, das die Übersetzung irgendwie komisch klingt, wenn man nicht auch den deutschsprachigen Lesefluss beachtet.

Reply:
Ich finde "Lesefluss" ist ein sehr subjektives Argument. Ich fühle mich in meinem hier nicht gestört, konsumiere aber seit Jahren mehr englisches als deutsches und bin daher vermutlich kein guter Maßstab mehr. 
Kann bleiben wie es ist.